src/plugins/console/public/lib/es/es.ts

```diff
@@ -39,17 +42,34 @@ export async function send({
   withProductOrigin = false,
   asResponse = false,
 }: SendProps) {
-  const options: HttpFetchOptions = {
+  const isKibanaApiRequest = path.includes(KIBANA_API_KEYWORD);
+
+  if (isKibanaApiRequest) {
+    const httpMethod = method.toLowerCase() as Method;
+    const uri = new URL(
+      `${window.location.origin}/${trimStart(path.replace(KIBANA_API_KEYWORD, ''), '/')}`
```

Comment:
I think I understand the logic of using `window.location.origin` here, but could you please add a short comment to clarify?